cfgov/unprocessed/apps/tccp/css/main.less

```diff
@@ -93,8 +93,21 @@
 }
 
 .o-well.o-well--speed-bump {
-  padding: unit((20px / @base-font-size-px), rem);
+  padding: unit((75px / @base-font-size-px), rem)
+    unit((35px / @base-font-size-px), rem);
   background-color: var(--teal-10);
+
+  .cf-icon-svg {
+    color: var(--green);
+  }
+
+  // Make everything bigger on desktop
+  .respond-to-min(@bp-sm-min, {
+    p {
+      .h3();
```

Comment:
Is there a way to make the "Did you know" have a weight of 500 even on desktop? Currently it looks like this:

|Mobile|Desktop|
|-|-|
|<img width="461" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/b98fe66c-48c5-45fc-9219-6873690b2307">|<img width="1020" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/862d7f78-ff6c-4ae4-9138-791db0e1601b">|

Reply:
The `.h3()` mixin steps down a size at mobile via a [media query](https://github.com/cfpb/design-system/blob/main/packages/cfpb-core/src/base.less#L236). Check that the CSS isn't ending up with a nested media query here. If you don't need the step-down behavior, you can use `.heading-3()` instead. If you find you're overriding things (such as for having the 500 weight across screen sizes), I would suggest copying the needed h3 styles into here and leaving a comment above it that says something like `// h3 sizing.`

Reply:
The lighter speed bump heading on desktop is by design. Per the designs, the heading is supposed to behave like an h3 across desktop and mobile while the speed bump body is h3 only at desktop. @anselmbradford that's a great tip about using `.heading-3()` to avoid the nested media query.